chat/management/commands/populate_messages.py

```diff
@@ -0,0 +1,52 @@
+from django.core.management.base import BaseCommand
+from users.documents import UserDocument, UserRoleDocument, UserRoleEnum
+from chat.documents import Room, Message
+from random import choice
+
+TEST_USERS = [
+    {"email": "user1@example.com", "first_name": "User", "last_name": "One"},
+    {"email": "user2@example.com", "first_name": "User", "last_name": "Two"},
+    {"email": "user3@example.com", "first_name": "User", "last_name": "Three"},
+]
+
+MESSAGES = [
+    "Hello everyone!",
+    "How are you?",
+    "All good, thanks!",
+    "What's new?",
+    "Who’s here?"
+]
+
+
+class Command(BaseCommand):
+    help = "Populate MongoDB with test users, rooms and messages"
+
+    def handle(self, *args, **kwargs):
+        role_user = UserRoleDocument.objects(role=UserRoleEnum.USER).first()
+        if not role_user:
+            role_user = UserRoleDocument(role=UserRoleEnum.USER).save()
+
+        users = []
+        for u in TEST_USERS:
+            user = UserDocument.objects(email=u["email"]).first()
+            if not user:
+                user = UserDocument(
+                    email=u["email"],
+                    first_name=u["first_name"],
+                    last_name=u["last_name"],
+                    password="pass123",
+                    role=role_user,
+                    is_active=True,
+                ).save()
+            users.append(user)
```

Comment:
Hardcoded password "pass123" is insecure. Consider using a secure password or making it configurable. Add a comment warning not to use this in production.
If your UserDocument hashes passwords, ensure this is respected; otherwise, users may have unusable credentials.